### PR TITLE
Add support to the single connection Interleaving in Search API

### PR DESCRIPTION
**Pull request by yiwenwang-ms**
Add support to the single connection Interleaving in Search API

### CODE REVIEW

File: concepts/search-concept-interleaving.md [hunk 1]
```diff
@@ -132,7 +132,118 @@ Content-type: application/json
 }
```
Comment (JarbasHorst) [inline concepts/search-concept-interleaving.md:132]: Hi @yiwenwang-ms, in row 36, is it correct to say "any connector combination" as the search API searches with *? 

- `### Example 1: Search with SharePoint file types and any connector combination`

**Comment by JarbasHorst** [comment]
Hello @yiwenwang-ms, I reviewed your PR and left one comment. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [21923](https://github.com/microsoftgraph/microsoft-graph-docs/pull/21923) into your branch to include my fixes and before addressing my comments.

File: concepts/search-concept-interleaving.md (hunk 1 above)
Comment (yiwenwang-ms) [reply concepts/search-concept-interleaving.md:132]: Thanks, I think I want to say all connector combination, just added the 'all'.

**Comment by JarbasHorst** [comment]
Hi @Lauragra, this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Review by njerigrevious — APPROVED** [review]
Looks great - thank you for the edits!

**Pull request merged**